### PR TITLE
GPG-651 Content changes for Manage employers journey

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
Content changes for the Manage Employers journey.

The content is a lot more similar between the In/Out change screens, so I've done some refactoring to bring them together. There are a lot of organisation routes here (and a lot of classes/files with Organisation) that I think it would probably be better to just cover when we do the ticket to change all of them (eventually).

### CODE REVIEW

**Comment by AlexJacksonDS** [comment]
Merged main into the branch to resolve merge conflicts

File: GenderPayGap.WebUI/Views/Scope/OutOfScopeQuestions.cshtml [hunk 1]
```diff
@@ -99,23 +99,11 @@
...
-                                       (opens in a new window)
-                                   </span>
-                               </a>
-                            </text>
-                }
-                ))
+                }))
+            
+            <p class="govuk-body">
+                Read  more about <a href="@Global.WhoNeedsToReportGuidanceLink" target="_blank" rel="noopener">who needs to report their gender pay gap.</a>
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/Scope/OutOfScopeQuestions.cshtml:105]: I noticed all the links that open a new tab have `<span class="govuk-visually-hidden">(opens in a new window)</span>` and I think we should add it here as well

File: GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml [hunk 2]
```diff
@@ -48,78 +48,23 @@
...
+            <a class="button" href="@Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = encryptedOrganisationId})" role="button">Return to @Model.Organisation.OrganisationName</a>
         </p>
-        
         <p class="govuk-body">
-            Return to
-            <a href="@Url.Action("ManageOrganisationsGet", "ManageOrganisations")"
-               class="govuk-link">
-                Your Organisations
-            </a>
+            <a class="button" href="@Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = ViewContext.RouteData.Values["Id"]})" role="button">Return to Manage Employers</a>
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml:67]: Should we use `govuk-button` instead? The ticket says "all components up to date as per GDS design system (keep an eye on the ‘details’ component, button styles " (I _think_ that class is up to date?)?

File: GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml [hunk 3]
```diff
@@ -20,10 +20,10 @@
 @section BeforeMain {
     @{
         var crumbs = new List<CrumbViewModel>
-        {
+{
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml:23]: is this by mistake?

File: GenderPayGap.WebUI/Views/Organisation/ConfirmRemove.cshtml [hunk 4]
```diff
@@ -1,21 +1,51 @@
...
+                Text = "Remove user"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/Organisation/ConfirmRemove.cshtml:39]: The crumbs seem to be the same for all the pages - the only thing that differs is the text for the last one. I'd add a separate component and use it everywhere

**Review by malinaslevoaca1 — COMMENTED** [review]
Nice work! A few suggestions:

File: GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml (hunk 2 above)
Comment (AlexJacksonDS) [reply GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml:67]: I've actually put completely the wrong link here, and I think they should be govuk-link according to the designs in the PDF

File: GenderPayGap.WebUI/Views/Organisation/ConfirmRemove.cshtml (hunk 4 above)
Comment (AlexJacksonDS) [reply GenderPayGap.WebUI/Views/Organisation/ConfirmRemove.cshtml:23]: Can't fully pull all the repeated content into the same partial, but have pulled as much as possible, I guess it's slightly better?

File: GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml (hunk 3 above)
Comment (AlexJacksonDS) [reply GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml]: Yep copy formatting error

**Review by malinaslevoaca1 — DISMISSED** [review]

**Comment by AlexJacksonDS** [comment]
Rebased on main to resolve conflicts preventing automatic rebase and merge

File: GenderPayGap.WebUI/Views/Shared/Patterns/ManageOrganisationBreadcrumbs.cshtml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            Text = Model.PageText
+        }
+    };
+}
+
+@(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+{
+    Crumbs = crumbs
+}))
```
Comment (malinaslevoaca1) [inline GenderPayGap.WebUI/Views/Shared/Patterns/ManageOrganisationBreadcrumbs.cshtml:28]: Can we use this on the reporting pages as well?

**Review by malinaslevoaca1 — APPROVED** [review]
Great work 👍

**Pull request merged**